02-library/cs506/sim.py

```
@@ -1,13 +1,56 @@
+import math
 def euclidean_dist(x, y):
-    raise NotImplementedError()
+    if len(x) != len(y):
+        return "Vectors have different dimention"
+    else:
+        sum = 0
+        for i in range(len(x)):
+            sum = sum + (x[i] - y[i])*(x[i] - y[i])
+        distance = math.sqrt(sum)
+        return distance
 
 def manhattan_dist(x, y):
-    raise NotImplementedError()
+    if len(x) != len(y):
+        return "Vectors have different dimention"
```

you should raise an error